### DISCUSSION TITLE
Path test coverage

### DISCUSSION THREAD

**Discussion by rostockahoi** [hotmeteor/spectator#117] [Show and tell]
Hi there,

I just wanted to share an idea that I had the other day: I am using Stoplight to create the Open API Spec for a Laravel API app. I integrated the Spec into Laravel's test suite using Spectator.

Now, at one point I got really lost about which paths/response codes I already covered with tests and which not. So I cooked up an Artisan command that would analyse all the paths and their response codes in the Spec and would compare them with the actual test methods that are present in the test suite.

Let me know what you think. Maybe it's worth to create a package.

This is how the HTML report looks like:

![api-path-coverage html](https://user-images.githubusercontent.com/16897125/184887497-49853e46-4874-4f7b-a8d1-6789cc462f37.png)

**Comment by jarrodparkes** [comment]
Love the look of this 😄 !
